templates/search/blocks/search_results_timeline_century_orig.html

```diff
@@ -0,0 +1,137 @@
+<html lang="en">
+<head>
+  <script src="https://cdn.jsdelivr.net/npm/chart.js@3.9.1"></script>
+
+  <script>
+
+    // Create an array of century labels (0 - 21)
+    const centuryLabels = [];
+    for (let century = 0; century <= 21; century++) {
+      centuryLabels.push(century + "00");
+    }
+
+    // Function to process data (adapt based on data structure)
+    function processData(data) {
+      const centuryCounts = {};
+      for (const item of data.data) {
+        // Check for missing data in access path
+        if (!item["@template"] || !item["@template"].details || !item["@template"].details.creationDateFrom) {
+          continue; // Skip entries with missing data
+        }
+
+        const creationFrom = parseInt(item["@template"].details.creationDateFrom, 10);
+
+        // Check for invalid creation date
+        if (isNaN(creationFrom)) {
+          continue; // Skip entries with invalid creation dates
+        }
+
+        const century = Math.floor(creationFrom / 100) * 100;
+
+        centuryCounts[century] = (centuryCounts[century] || 0) + 1;
+      }
+
+      // Convert centuryCounts to data for chart
+      const centuryData = new Array(22).fill(0);
+      for (const [century, count] of Object.entries(centuryCounts)) {
+        const centuryIndex = Math.floor(century / 100);
+        centuryData[centuryIndex] = count;
+      }
+
+      return centuryData;
+    }
+
+    // Fetch the JSON data from a URL (if applicable)
+
+    fetch("/static/timeline/timeline.json")
```

Comment:
Above is an exclusive API call, then is the static file necessary? Also called from other places.

Reply:
This is a back up file in case I need to revert, it will be removed eventually.  search_results_timeline_century_**orig.**html

Reply:
> This is a back up file in case I need to revert, it will be removed eventually. search_results_timeline_century_**orig.**html

I think its easier as its a whole file to visit this file from history when its removed. Alternatively do you want to note a clean-up step in the ticket?